packages/eservice-template-process/test/integration/updateEServiceTemplateRiskAnalysis.test.ts

```diff
@@ -69,8 +67,10 @@ describe("updateEServiceTemplateRiskAnalysis", () => {
       kind: creatorTenantKind,
     };
 
-    const riskAnalysis1 = getMockValidRiskAnalysis(creatorTenantKind);
-    const riskAnalysis2 = getMockValidRiskAnalysis(creatorTenantKind);
+    const riskAnalysis1 =
+      getMockValidEServiceTemplateRiskAnalysis(creatorTenantKind);
+    const riskAnalysis2 =
+      getMockValidEServiceTemplateRiskAnalysis(creatorTenantKind);
```

Comment:
if you want to make this const more redeable and clear, we can using explicative name, for example:
 
```suggestion
    const riskAnalysisToUpdate =
      getMockValidEServiceTemplateRiskAnalysis(creatorTenantKind);
    const riskAnalysisSecondary =
      getMockValidEServiceTemplateRiskAnalysis(creatorTenantKind);
```